src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -60,6 +66,12 @@
 	private int readinessProbeDelay = 10;
 
 	/**
+	 * Period in seconds to perform the readiness check of the app container.
+	 */
+	// see http://kubernetes.io/v1.0/docs/user-guide/production-pods.html#liveness-and-readiness-probes-aka-health-checks}
+	private int readinessPeriod = 10;
```

Comment:
For consistency, this should be named `readinesProbePeriod` (just as 'probe' is used elsewhere).